tests/cli/test_root.py

```diff
@@ -50,3 +50,52 @@ def test_resource_presets_uninitialized(root_uninitialized: Root) -> None:
 
 def test_get_session_cookie(root_uninitialized: Root) -> None:
     assert root_uninitialized.get_session_cookie() is None
+
+
+class TestTokenSanitization:
+    @pytest.mark.parametrize(
+        "auth", ["Bearer", "Basic", "Digest", "Mutual"],
+    )
+    def test_sanitize_header_value_single_token(
+        self, root_uninitialized: Root, auth: str
+    ) -> None:
+        line = f"foo Authentication: {auth} eyJhbGciOiJI.eyJzdW0NTY3.SfKxwRJ_SsM bar"
+        expected = f"foo Authentication: {auth} eyJhb<hidden 26 chars>J_SsM bar"
```

Comment:
`_sanitize_header_value` is for the value by definition, a test for `key: value` string is misleading.

The same below.

Reply:
my bad, fixing.